src/main/java/com/javadeobfuscator/deobfuscator/transformers/special/ParamorphismTransformer.java

```diff
@@ -359,6 +346,37 @@ else if(replacements.get(caller) instanceof FieldInsnNode)
 				}
 			}
 		}
+		classNodes().stream().parallel().forEach(cn -> {
+			cn.methods.stream().parallel().forEach(mn -> {
+				for (AbstractInsnNode ain : mn.instructions.toArray()) {
+					if (ain instanceof MethodInsnNode && !((MethodInsnNode) ain).owner.startsWith("java/lang/")) {
```

Comment:
Need a quick reminder (it's been awhile since I have seen Paramorphism):  What exactly is the check for "java/lang/" for?

Reply:
I guess the idea is to only replace method calls to non-library functions.

Reply:
Yes. Ignore lib-calls. It only decrypt the redirected calls.